src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -15,7 +15,7 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
+    public static final String USERGUIDE_URL = "https://nus-cs2103-ay2223s1.github.io/tp/UserGuide.html";
```

Comment:
I think the url for the UG should be https://ay2223s1-cs2103t-t12-1.github.io/tp/UserGuide.html